src/test/java/org/broadinstitute/hellbender/utils/nio/NioTests.java

```diff
@@ -0,0 +1,118 @@
+package org.broadinstitute.hellbender.utils.nio;
+
+import com.google.cloud.AuthCredentials;
+import com.google.cloud.RetryParams;
+import com.google.cloud.storage.StorageException;
+import com.google.cloud.storage.StorageOptions;
+import com.google.cloud.storage.contrib.nio.CloudStorageConfiguration;
+import com.google.cloud.storage.contrib.nio.CloudStorageFileSystem;
+import org.broadinstitute.hellbender.utils.test.BaseTest;
+import org.testng.annotations.Test;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URI;
+import java.nio.file.*;
+import java.util.Arrays;
+
+/**
+ * Test GCS access via the NIO APIs.
+ */
+public final class NioTests extends BaseTest {
+
+    @Test
+    public void testGcsEnabled() {
+        FileSystem fs = FileSystems.getFileSystem(URI.create("gs://domain-registry-alpha"));
+    }
+
+    @Test
+    public void openPublicFile() throws IOException {
+        try (InputStream inputStream = Files.newInputStream(Paths.get(URI.create(
+                "gs://pgp-harvard-data-public/hu011C57/GS000018120-DID/GS000015172-ASM/manifest.all.sig")))) {
```

Comment:
I would say it pretty much is, yes. But feel free to suggest a different file if you prefer!